<launch>
  <param name="result_dir" type="str" value="$(find hexapod_training)/results" />
   <node pkg="hexapod_training" name="q_plotter" type="QPlotter.py" output="screen"/>
</launch>
